po/it/system-monitor.po

```diff
@@ -207,7 +207,7 @@ msgid "other"
 msgstr "altro"
 
 msgid "nice"
-msgstr "nice"
```

Comment:
AFAIK, "nice" is a technical term and isn't translated in Italian. I'm Italian, but I never heard of "particolare" as a translation for "nice"; can you point to a reference for that?